pkg/azureclients/diskclient/interface.go

```diff
@@ -19,7 +19,7 @@ package diskclient
 import (
 	"context"
 
-	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2021-12-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2022-03-01/compute"
```

Comment:
The APIVersion in L29 is "2022-03-02", @andyzhangx which is the correct one we should use?

Reply:
apiVersion for disk is correct, it's `2022-03-02`, the sdk path is just `2022-03-01`, that's not related.
https://learn.microsoft.com/en-us/azure/templates/microsoft.compute/2022-03-02/disks